R/html_document.R

```diff
@@ -22,8 +22,10 @@
 #'  options that control the behavior of the floating table of contents. See the
 #'  \emph{Floating Table of Contents} section below for details.
 #'@param number_sections \code{TRUE} to number section headings
-#'@param anchor_sections \code{TRUE} to show section anchors when mouse hovers.
-#'  See \link[rmarkdown:html_document]{Anchor Sections Customization section}.
+#'@param anchor_sections \code{TRUE} to show section anchors when mouse hovers
+#'  for all headers. A list can also be passed with \code{style} and/or
+#'  \code{depth} to customize the behavior. See
+#'  \link[rmarkdown:html_document]{Anchor Sections Customization section}.
```

Comment:
I added that because if case other formats import params for `html_document`. Only this text would be imported but not the below section. So you would need to know to look there for futher details. The link seemed useful in this case.

Not a good idea ?

Reply:
Oh yeah, we don't have a good solution for this case 😢

Reply:
This links I added works to move to `html_document` help page but it does not show the section directly, you would still have to look down to find it. 

Documenting in section is useful to describe our formats' features but we have this difficulties to navigate to them. Maybe I should spend time on vignettes instead, like one per format, that we could link to.

Do you think that would be better than my solution ?

Reply:
There's no great way to link to a section within a vignette either; except to link to link to the html on the rendered website (which you could also if it's in function documentation). In general, I don't have a great sense for when you should use function documentation vs a vignette. The primary disadvantage of a vignette is that it's quite far from the function so its easy to forget to update it.